app/views/coronavirus_landing_page/components/landing_page/_arrow.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if number.positive? %>
```

Comment:
Do we need a 3rd case here for 0? Otherwise I assume we're going to get a down arrow for it which seems wrong

Reply:
Perhaps the conditional should be the other way around? i.e. check for `negative?` to show a down arrow and show an up arrow for everything else?  Or change the conditional to also check for `number.zero?`? 

Do we even want to so an arrow for zero?

What do you think @andysellick? Maybe add a note to the commit message about `positive?` and `negative?` both returning `false` for zero.

Reply:
I've changed this to an `if / elsif` so that if it's ever `0` we won't get an arrow shown at all.

Reply:
thanks :)